### PR TITLE
update examples/limitrange to v1beta3

### PR DESCRIPTION
**Pull request by caesarxuchao**
- Update examples/limitrange to v1beta3.
- Change the Memory req. in invalid-pod to 5Mi. The old value is 1Mi, which is not valid even before the limitrange resource is created, because the default minimum is 4Mi.
- Add a minimum readme.

@nikhiljindal @derekwaynecarr

### CODE REVIEW

**Comment by nikhiljindal** [comment]
LGTM

File: examples/limitrange/invalid-pod.json [hunk 1]
```diff
@@ -1,20 +1,22 @@
 {
-  "id": "invalid-pod",
+  "apiVersion":"v1beta3",
   "kind": "Pod",
-  "apiVersion":"v1beta2",
+  "metadata": {
+    "name": "invalid-pod"
+  },
   "labels": {
```
Comment (bgrant0607) [inline examples/limitrange/invalid-pod.json]: labels belong in metadata

File: examples/limitrange/valid-pod.json [hunk 2]
```diff
@@ -1,20 +1,22 @@
 {
-  "id": "valid-pod",
+  "apiVersion":"v1beta3",
   "kind": "Pod",
-  "apiVersion":"v1beta2",
+  "metadata": {
+    "name": "valid-pod"
+  },
   "labels": {
```
Comment (bgrant0607) [inline examples/limitrange/valid-pod.json]: labels belong in metadata

File: examples/limitrange/valid-pod.json [hunk 3]
```diff
@@ -1,20 +1,22 @@
...
-        "image": "gcr.io/google_containers/serve_hostname",
-        "cpu": 1000,
-        "memory": 5242880
-      }]
-    }
+  "spec": {
+    "containers": [{
+      "name": "kubernetes-serve-hostname",
+      "image": "gcr.io/google_containers/serve_hostname",
+        "resources": {
```
Comment (bgrant0607) [inline examples/limitrange/valid-pod.json]: indentation

Comment (caesarxuchao) [inline examples/limitrange/valid-pod.json]: Thanks. Updated.

**Comment by derekwaynecarr** [comment]
LGTM

**Pull request merged**